### PR TITLE
KOGITO-1138 Add node cancel action in process instance timeline

### PR DESCRIPTION
**Pull request by AjayJagan**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

description: added a node cancel option to the any active nodes in the timeline.

JIRA: https://issues.redhat.com/browse/KOGITO-1138

### CODE REVIEW

File: ui-packages/packages/management-console/src/components/Organisms/ProcessDetailsTimeline/ProcessDetailsTimeline.tsx [hunk 1]
```diff
@@ -73,23 +73,139 @@ const ProcessDetailsTimeline: React.FC<IOwnProps> = ({
...
 
-  const onKebabToggle = isOpen => {
-    setIsKebabOpen(isOpen);
+  const nodeManagementDropdownItems = [
+    <DropdownItem
+      key="cancel"
+      component="button"
+      onClick={() =>
+        handleNodeInstanceCancel(
+          data,
```
Comment (cristianonicolai) [inline ui-packages/packages/management-console/src/components/Organisms/ProcessDetailsTimeline/ProcessDetailsTimeline.tsx:90]: @AjayJagan I still not sure why we need to pass functions to show the popup. Ideally we should only pass the data that is needed here.

File: ui-packages/packages/management-console/src/utils/Utils.tsx [hunk 2]
```diff
@@ -204,3 +200,36 @@ export const modalToggle = (
...
+) => {
+  setModalTitle('Node cancel process');
+  axios
+    .delete(
+      `${processInstanceData.serviceUrl}/management/processes/${processInstanceData.processId}/instances/${processInstanceData.id}/nodeInstances/${nodeObject.id}`
+    )
+    .then(() => {
+      setTitleType('success');
+      setModalContent(
+        `The node - ${nodeObject.name} was successfully cancelled`
```
Comment (cristianonicolai) [inline ui-packages/packages/management-console/src/utils/Utils.tsx:220]:
```suggestion
        `The node ${nodeObject.name} was successfully cancelled`
```

Comment (srambach) [reply ui-packages/packages/management-console/src/utils/Utils.tsx:220]: This wording seems fine to me. However, my sources say the Red Hat standard is North American spellings, so we should use "canceled" with one "l"

**Comment by AjayJagan** [comment]
Closing this and creating a new PR with the one rebased with KOGITO-1138.

**Pull request closed without merging**